src/sentry/web/api.py

```diff
@@ -896,6 +908,11 @@ def post(self, request, project, **kwargs):
                 merge_attached_breadcrumbs(file.open_stream(), event)
                 continue
 
+            if file.type == "minidump":
+                if refactor_enabled:
+                    write_minidump_placeholder(event)
```

Comment:
I fixed it now. There are some issues wrt how apple crash reports are stored as attachments. The most obv one is that they are stored as attachment type = minidump. We need to invent a separate attachment type for those, but I'd like to avoid fixing that for now.

I added tests that snapshot ingested apple crash report events and made sure the only diff is the added unreal event context.